.github/ISSUE_TEMPLATE/release-tracking-issue.md

```diff
@@ -6,64 +6,77 @@ labels: release
-  - [ ] [kubernetes-sigs/aws-iam-authenticator](https://github.com/kubernetes-sigs/aws-iam-authenticator/releases)
   - [ ] etcd-io/etcd
+#### EKS-D
+  - [ ] [kubernetes-sigs/metrics-server](https://github.com/kubernetes-sigs/metrics-server/releases)
   - [ ] [kubernetes-csi/external-attacher](https://github.com/kubernetes-csi/external-attacher/releases)
   - [ ] [kubernetes-csi/external-provisioner](https://github.com/kubernetes-csi/external-provisioner/releases)
   - [ ] [kubernetes-csi/external-resizer](https://github.com/kubernetes-csi/external-resizer/releases)
   - [ ] [kubernetes-csi/external-snapshotter](https://github.com/kubernetes-csi/external-snapshotter/releases)
   - [ ] [kubernetes-csi/livenessprobe](https://github.com/kubernetes-csi/livenessprobe/releases)
   - [ ] [kubernetes-csi/node-driver-registrar](https://github.com/kubernetes-csi/node-driver-registrar/releases)
-- [ ] Check for patch changes
+
+### Check for patch changes (Mon/Tues)
+  - [ ] Check for patches for all versions
+
+### Start Code Freeze (Tues)
+- [ ] Announce code freeze for Non-Release Related Changes
+
+### Recheck for new PRs (Wed)
 - [ ] After all the release-related changes are done, recheck for new PRs created by `eks-distro-pr-bot` in the below repos; review and merge them
   - [ ] [eks-distro](https://github.com/aws/eks-distro/pulls/eks-distro-pr-bot)
   - [ ] [eks-distro-build-tooling](https://github.com/aws/eks-distro-build-tooling/pulls/eks-distro-pr-bot)
   - [ ] [eks-distro-prow-jobs](https://github.com/aws/eks-distro-prow-jobs/pulls/eks-distro-pr-bot)
 
-### Cut Release
-- [ ] Open and merge dev release PR
-- [ ] Confirm [Prow jobs](https://prow.eks.amazonaws.com/?repo=aws%2Feks-distro&type=postsubmit&job=*1-2*) `dev-release-1-XX-postsubmit` and `build-1-XX-postsubmit` finished successfully
-- [ ] Open and merge prod release PR
+### *Duplicate the following sections for each branch being released*:
+
+---
+
+{RELEASE_BRANCH}
+
+### Cut Dev Release (Wed AM)
+- [ ] Open and merge dev release PR {RELEASE_BRANCH}
+- [ ] Confirm [Prow jobs](https://prow.eks.amazonaws.com/?repo=aws%2Feks-distro&type=postsubmit&job=*1-2*) `dev-release-{RELEASE_BRANCH}-postsubmit` and `build-{RELEASE_BRANCH}-postsubmit` finished successfully
+
+### Cut Prod Release (Thurs AM)
+- [ ] Open and merge prod release PR {RELEASE_BRANCH}
 - [ ] Confirm prod release was successful
-  - [ ] [Prow jobs](https://prow.eks.amazonaws.com/?repo=aws%2Feks-distro&type=postsubmit&job=*1-2*) `prod-release-1-XX-postsubmit` and `build-1-XX-postsubmit` finished successfully
-  - [ ]  Release manifest is available and looks right (`https://distro.eks.amazonaws.com/kubernetes-1-XX/kubernetes-1-XX-eks-Z.yaml`)
+  - [ ] [Prow jobs](https://prow.eks.amazonaws.com/?repo=aws%2Feks-distro&type=postsubmit&job=*1-2*) `prod-release-{RELEASE_BRANCH}-postsubmit` and `build-{RELEASE_BRANCH}-postsubmit` finished successfully
+  - [ ] Release manifest is available and looks right
```

Comment:
note: it'd be nice to have an automated check, even just a simple script, that checks this